webrtc-c/canary/src/CanaryWebrtc.cpp

```diff
@@ -149,48 +172,39 @@ STATUS onNewConnection(Canary::PPeer pPeer)
     return retStatus;
 }
 
-VOID sendLocalFrames(Canary::PPeer pPeer, MEDIA_STREAM_TRACK_KIND kind, const std::string& pattern, UINT64 frameCount, UINT32 frameDuration)
+VOID sendCustomFrames(Canary::PPeer pPeer, MEDIA_STREAM_TRACK_KIND kind, UINT64 dataRate, UINT64 frameRate)
```

Comment:
I think we should preserve both of the functions. We might still want to get the pseudo-video frames with some combination of params to the canary